README.md

```diff
@@ -1 +1,84 @@
-# Portfolio
\ No newline at end of file
+<a name="readme-top"></a>
+
+<div align="center">
+
+  <img src="murple_logo.png" alt="logo" width="140"  height="auto" />
+  <br/>
+
+</div>
+
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+- [💻 Getting Started](#getting-started)
+- [Setup](#setup)
+- [Run tests](#run-tests)
+- [Deployment](#triangular_flag_on_post-deployment)
+- [👥 Authors](#authors)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [📝 License](#license)
+
+# Portfolio-setup-and-mobile-first
+
+> This is a simple replica of the https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=48%3A27&t=9GufdfER8lvr1MpO-0  template provided in the Microverse project. 
+
+Live link
+Follow this link to visit my Portfolio.
+http://127.0.0.1:5500/index.html
+
+![portfolio](https://user-images.githubusercontent.com/110582192/205082549-5c859179-5760-45d5-9a92-59514ba6df62.png)
+
+
+## Built With
+
+- HTML, CSS3
+- VSCode
+
+## Getting Started
+
+Clone the repository to get start with project, then make sure to install dependencies in the linters file located in the [linter](https://github.com/Bateyjosue/linters-html-css/blob/main/.github/workflows/linters.yml) file
+
+
+### Setup
+-Clone the folder to your local machine
+-Open the index.html file in your brower.
+
+
+### Run tests
+After pushing the changes on github go in code tab click on Merge pull request and let github action proess and check for changes.
+
+### Deployment
+Check for the tests when you generate a pull request and fix the errors if any.
+For stylelint error  run:<code>sudo npx stylelint "**/*.{css,scss}" --fix</code> and it will the fix style issues automatically.
+
+## Authors
+
+👤 **Ashmal Zahra**
+
+- GitHub: [@githubhandle](https://github.com/ashmalzahra)
+- Twitter: [@twitterhandle](https://twitter.com/AshmalZahraa)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/ashmal-zahra-35bb09242/)
+
+## 🤝 Contributing
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](https://github.com/ashmalzahra/Portfolio/issues).
+
+## Show your support
+
+Give a ⭐️ if you like this project!
+
+## Acknowledgments
+
+- Hat tip to anyone whose code was used
+- Inspiration
+- etc
```

Comment:
- [ OPTIONAL ] Please try to add an acknowledgments section as it helps to credit the right person or article in the correct way 💯